boards/posix/doc/bsim_boards_design.rst

```diff
@@ -242,13 +253,14 @@ which relies on the bs_trace API. Instead, for tracing the bs_trace API
 should be used directly.
 The same applies to other Zephyr APIs, including the entropy API, etc.
 
-posix_print backend
-===================
+posix_print and nsi_print backends
+==================================
 
 The bsim board provides a backend for the posix_print API which is expected by the posix ARCH
-and soc inf (POSIX) code.
-It simply routes the printk strings to the bs_trace bsim API.
-Any message printed to the posix_print API, which is also the default printk backend,
+and soc inf (POSIX) code, and for the nsi_print API expected by the native simulator.
```

Comment:
Above you write `soc_inf` instead of `soc inf`. Which is the correct one?

Reply:
All instances renamed to the actual SOC name :)